src/sentry/api/endpoints/team_projects.py

```diff
@@ -14,6 +14,9 @@
 
 ERR_INVALID_STATS_PERIOD = "Invalid stats_period. Valid choices are '', '24h', '14d', and '30d'"
 
+# This will be used when no project name is provided
+DEFAULT_NEW_PROJECT_NAME = 'new-project'
```

Comment:
Maybe this could live in a constants module somewhere? Where are we usually putting defaults?